wedrix/watchtower-bundle/1.0/config/packages/wedrix_watchtower_bundle.yaml

```diff
@@ -0,0 +1,18 @@
+wedrix_watchtower:
+    endpoint: '/graphql.json'
+    schema_file: '%kernel.project_dir%/config/watchtower/schema.graphql'
+    plugins_directory: '%kernel.project_dir%/config/watchtower/plugins'
+    scalar_type_definitions_directory: '%kernel.project_dir%/config/watchtower/scalar_type_definitions'
+    schema_cache_directory: '%kernel.project_dir%/var/cache/watchtower/schema'
+    cache_schema: false
+    debug: true
```

Comment:
Minor - but this could be set in the. bundle, automatically, to the debug mode of the app.